---
# Feel free to add content and custom Front Matter to this file.
# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults

layout: home
---


### Introduction

Despite having the highest genomic diversity of any continent, African populations are the least studied in the context of human genetic variation. 

The African Genome Variation Database (AGVD) aims to maximize the utility of available variant data gathered from studies of African cohorts and other cohorts with African-ancestry. The initial release reports allele frequencies from a joint called set of ~4000 samples. Future releases will incorporate genotypes from targeted, exome, and additional full genome sequencing, as well as genotyping by arrays. 

We expect researchers focusing on African populations or diseases to find novel value in the AGVD as a resource, not only through the search and visualization features but also from its focus on presenting African data. 

AGVD's primary objectives are to:
 - Maximize benefit from and access to variant frequencies from African populations.
 - Provide search and visualization capabilities, enabling researchers to explore genetic data by study, ethnic group, and specific variants. 
 - Provide insights into the clinical relevance of SNPs by presenting annotations from internal and external sources. 
 - Allow more targeted search in terms of allele frequencies, eg rare disease searches.
 - Promote interoperability with other projects by adopting GA4GH standards such as Beacons, VRS, and Passport

Development of the AGVD was funded by H3ABioNet, a project within the H3Africa Consortium. 

 - AGVD is accessible at [agvd.h3abionet.org](https://agvd.h3abionet.org/)
 - A comprehensive list of contributors can be found on [this page](contributors.md).
 - [Release notes](releases.md)
 - [Sampling locations](map.md)
 

H3ABioNet is funded by NIH Common Fund Award / NHGRI Grant Number U41HG006941.

<footer><p><i>this documentation was build at {{ site.time }}</i></p></footer>